agent/policyMgr/manager.go

```diff
@@ -75,6 +77,17 @@ func (a *policyManager) ManagePolicy(payload fleet.AgentPolicyRPCPayload) {
 					a.logger.Warn("policy failed to ensure dataset id", zap.String("policy_id", payload.ID), zap.String("policy_name", payload.Name), zap.String("dataset_id", payload.DatasetID), zap.Error(err))
 				}
 			}
+			// if policy already exist and has no version upgrade, has no need to apply it again on pktvisor
+			currentPolicy, err := a.repo.Get(payload.ID)
+			if err != nil {
+				a.logger.Error("failed to retrieve policy", zap.String("policy_id", payload.ID), zap.Error(err))
+			}
+			if currentPolicy.Version >= pd.Version {
+				a.logger.Info("policy already applied or with greater version, skipping", zap.String("policy_id", pd.ID), zap.String("policy_name", pd.Name), zap.String("attempted version", fmt.Sprint(pd.Version)), zap.String("current version", fmt.Sprint(currentPolicy.Version)))
```

Comment:
how about "we have a better version of this policy, skipping". the log keys should not have spaces, so "attempted_version" and "current_version"